tests/setup_nltephotospheric_dynamic_ion_range_1d_1dgrid.sh

```diff
@@ -6,21 +6,21 @@ runfolder=nltephotospheric_dynamic_ion_range_1d_1dgrid_testrun
 
 mkdir -p $runfolder
 
-if [ ! -f atomicdata_feconi.tar.xz ]; then curl -O https://theory.gsi.de/~lshingle/artis_http_public/artis/atomicdata_feconi.tar.xz; fi
+if [ ! -f atomicdata_hefeconi_fe_I_to_VII.tar.xz ]; then curl -O https://theory.gsi.de/~lshingle/artis_http_public/artis/atomicdata_hefeconi_fe_i_to_vii.tar.xz; fi
 
-tar -xf atomicdata_feconi.tar.xz --directory $runfolder
+tar -xf atomicdata_hefeconi_fe_i_to_vii.tar.xz --directory $runfolder
 
 rsync -av --ignore-times nltephotospheric_dynamic_ion_range_1d_1dgrid_inputfiles/ $runfolder
 
-rsync -av nebular_1d_3dgrid_inputfiles/abundances.txt nebular_1d_3dgrid_inputfiles/model.txt nebular_1d_3dgrid_inputfiles/recombrates.txt $runfolder
+rsync -av nebular_1d_3dgrid_inputfiles/model.txt $runfolder
```

Comment:
It's not new to this PR, but it looks like these lines got swapped around? Normally we'd copy the whole folder and the path it with updates, like this in setup_nebular_1d_3dgrid_limitbfest.sh:
```sh
rsync -av nebular_1d_3dgrid_inputfiles/ nebular_1d_3dgrid_limitbfest_testrun/
rsync --ignore-times -av nebular_1d_3dgrid_limitbfest_inputfiles/ nebular_1d_3dgrid_limitbfest_testrun/
```
For example, recombrates.txt was not copied and it's not clear whether this is intentional or not.